packages/core/src/Stores/traders-hub-store.js

```diff
@@ -380,6 +382,12 @@ export default class TradersHubStore extends BaseStore {
         this.setCombinedCFDMT5Accounts();
     }
 
+    get restricted_countries() {
+        const { financial_company, gaming_company } = this.root_store.client.landing_companies;
+
+        return gaming_company?.shortcode === 'svg' && !financial_company;
+    }
+
```

Comment:
isnt this same as `is_high_risk` in client store?

Reply:
For `is_high_risk` both gaming and financial shortcodes should have `svg` as their value @amina-deriv

Reply:
@matin-deriv @thisyahlen-deriv we have added the below condition for restricted countries condition to the `is_high_risk` inorder to cater Australia/Norway clients.

```
const restricted_countries =
            financial_company?.shortcode === 'svg' ||
            (gaming_company?.shortcode === 'svg' && financial_company?.shortcode !== 'maltainvest');
```

Reply:
The thing is that this is a very specific check @amina-deriv. Japan/Norway only has gaming company svg in the response. Very unique compared to the others. Do you have any suggestions ?

Reply:
oh ok. I see below comment from @sergei-deriv about restricted_countries on that is_high_risk function and thats why i checked will that same thing works here
// This is a conditional check for countries like **Australia/Norwa**y which fulfil one of these following conditions.